Tests/SwiftPhoenixClientTests/SocketSpec.swift

```diff
@@ -76,21 +76,29 @@ class SocketSpec: QuickSpec {
       
       it("should construct a valid URL", closure: {
         
+        // test vsn
+        expect(Socket("http://localhost:4000/socket/websocket",
+                      paramsClosure: { ["token": "abc123"] },
+                      vsn: "1.0.0")
```

Comment:
`vsn` can be supplied in `paramsClosure`, should it still need to be a separate argument?

Reply:
I would say yes since that is the API that the js library uses.